community-meetings/2022-01-07.md

```diff
@@ -0,0 +1,20 @@
+# Agenda for the GoReleaser Community Meeting - January 7, 2022
+
+## Links for the participants
+
+- YouTube live stream /recording: [Youtube](https://www.youtube.com/channel/UCxg5N16FKrTa4Cees434pbw)
+- Current Milestones: [v1.1.0](https://github.com/goreleaser/goreleaser/milestone/3)
```

Comment:
1.1 and 1.2 were released, maybe 1.3/1.4 or simply "next milestones"?

Reply:
Yes, should do